_episodes/07-github.md

```diff
@@ -112,22 +119,188 @@ $ git remote -v
+The key's randomart image is:
++--[ED25519 256]--+
+|^B== o.          |
+|%*=.*.+          |
+|+=.E =.+         |
+| .=.+.o..        |
+|....  . S        |
+|.+ o             |
+|+ =              |
+|.o.o             |
+|oo+.             |
++----[SHA256]-----+
+~~~
+{: .output}
+
+The "identification" is actually the private key. You should never share it.  The public key is appropriately named.  The "key fingerprint" 
+is a shorter version of a public key.
+
+Now that we have generated the SSH keys, we will find the SSH files when we check.
+
+~~~
+ls -al ~/.ssh
+~~~
+{: .language-bash}
+
+~~~
+drwxr-xr-x 1 Vlad Dracula 197121   0 Jul 16 14:48 ./
+drwxr-xr-x 1 Vlad Dracula 197121   0 Jul 16 14:48 ../
+-rw-r--r-- 1 Vlad Dracula 197121 419 Jul 16 14:48 id_ed25519
+-rw-r--r-- 1 Vlad Dracula 197121 106 Jul 16 14:48 id_ed25519.pub
+~~~
+{: .output}
+
+Now we run the command to check if GitHub can read our authentication.  
+
+~~~
+ssh -T git@github.com
+~~~
+{: .language-bash}
+
+
+~~~
+The authenticity of host 'github.com (192.30.255.112)' can't be established.
+RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
+This key is not known by any other names
+Are you sure you want to continue connecting (yes/no/[fingerprint])? y
+Please type 'yes', 'no' or the fingerprint: yes
+Warning: Permanently added 'github.com' (RSA) to the list of known hosts.
+git@github.com: Permission denied (publickey).
+~~~
+{: .output}
+
+Right, we forgot that we need to give GitHub our public key!  
+
+First, we need to copy the public key.  Be sure to include the `.pub` at the end, otherwise you’re looking at the private key. 
+
+~~~
+cat ~/.ssh/id_ed25519.pub
+~~~
+{: .language-bash}
+
+~~~
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDmRA3d51X0uu9wXek559gfn6UFNF69yZjChyBIU2qKI vlad@tran.sylvan.ia
+~~~
+{: .output}
+
+Now, going to GitHub.com, click on your profile icon in the top right corner to get the drop-down menu.  Click "settings," then on the 
+settings page, click "SSH and GPG keys," on the left side "Account settings" menu.  Click the "New SSH key" button on the right side. Now, 
+you can add the title (Dracula uses the title "Vlad's Lab Laptop" so he can remember where the original key pair
+files are located), paste your SSH key into the field, and click the "Add SSH key" to complete the setup.
+
+Now that we’ve set that up, let’s check our authentication again from the command line. 
+~~~
+$ ssh -T git@github.com
+~~~
+{: .language-bash}
+
+~~~
+Hi Vlad! You've successfully authenticated, but GitHub does not provide shell access.
+~~~
+{: .output}
+
+## Push local changes to a remote
+
+Now that authentication is setup, we can return to the remote.  This command will push the changes from
 our local repository to the repository on GitHub:
 
 ~~~
 $ git push origin main
 ~~~
 {: .language-bash}
 
+Since Dracula set up a passphrase, it will prompt him for it.  If you completed advanced settings for your authentication, it 
```

Comment:
```suggestion
Since Dracula set up a passphrase for his SSH key, git will prompt him for it.  If you completed advanced settings for your authentication, it 
```

Reply:
Technically, I don't think it's git that prompts for the passphrase, but SSH.